README.md

```diff
@@ -29,31 +29,90 @@ yarn
 ```
 
 ## Customize your deployment parameters
 
 You can customize all of the deployment parameters of the LSP simply by changing the parameters that you pass in the run command above. See [the script](./index.js) or [documentation](https://docs.umaproject.org/synthetic-tokens/long-short-pair#lsp-construction-parameters) for more details about these parameters.
 
-Your financial product library address will defines the payout function for your LSP. We have several [financial product libraries](https://github.com/UMAprotocol/protocol/tree/master/packages/core/contracts/financial-templates/common/financial-product-libraries/long-short-pair-libraries) available for transforming the price, identifier, or collateral requirement of an LSP before or after expiry. For these addresses, see the `Contract Addresses` section. In some cases, you may find yourself in need of a custom financial product library for your use case. If so, please see the `Deploying new financial product libraries`.
+Your financial product library address will defines the payout function for your LSP. We have several [financial product libraries](https://github.com/UMAprotocol/protocol/tree/master/packages/core/contracts/financial-templates/common/financial-product-libraries/long-short-pair-libraries) available for transforming the price, identifier, or collateral requirement of an LSP before or after expiry.
 
-Note that the `LongShortPairCreator` contract will differentiate between long and short tokens by appending " Long Token" or " Short Token" to `syntheticName` and prepending "l" or "s" to the `syntheticSymbol`.
+How and when to use various libraries is explained below.
 
-## Contract Addresses
+### Binary Option
 
-See the files linked below for the `LongShortPairCreator` and financial product library contract addresses on various networks. You can pass these as parameters when deploying.
+Binary options settle with all collateral allocated to either the long or short side, depending on the settlement price. They can be used to make prediction markets or any kind of binary bet. Settlement is defined using a strike price which informs which side of the bet was correct. If settlement price is greater or equal to the strike then all value is sent to the long side. Otherwise, all value is sent to the short side. The settlement price could be a scalar (like the price of ETH) or a binary bet with settlement being 0 or 1 depending on the outcome.
```

Comment:
Really low lift that would make this more actionable would just be noting how the params in the FPLs factor into these equations.